gpcontrib/arenadata_toolkit/src/include/bloom_set.h

```diff
@@ -0,0 +1,36 @@
+#ifndef BLOOM_SET_H
+#define BLOOM_SET_H
+
+#include "postgres.h"
+
+#include "bloom.h"
+
+#define FULL_BLOOM_ENTRY_SIZE(size) (offsetof(bloom_entry_t, bloom) + FULL_BLOOM_SIZE(size))
+#define FULL_BLOOM_SET_SIZE(size, count) (offsetof(bloom_set_t, bloom_entries) + FULL_BLOOM_ENTRY_SIZE(size) * count)
+
+/* Bloom set entry. */
```

Comment:
As `bloom_entry_t` and `bloom_set_t` are not a part of the bloom_set api, I suggest to move all these guts inside the bloom_set.c file. 

And `FULL_BLOOM_ENTRY_SIZE` , `FULL_BLOOM_SET_SIZE` as well. As `FULL_BLOOM_SET_SIZE` is used outside the bloom_set component in `tf_shmem_calc_size()`, I think it is better to change it to a function. smth like `bloom_set_get_required_mem_size` or whatsoever similar.